### ISSUE TITLE
Update to the latest version of IOSSecuritySuite

### ISSUE THREAD

**Issue by kostadin-damyanov-prime** [jeroentrappers/flutter_jailbreak_detection#63]
As you may already know, starting 1st of May Apple will require apps to have reason of use for some system APIs.
For reference: [the official docs](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api).

It looks like **IOSSecuritySuite** [recently added this](https://github.com/securing/IOSSecuritySuite/blob/master/IOSSecuritySuite/Resources/PrivacyInfo.xcprivacy) to their repo. I think that this package needs to be updated to the latest version of this dependency, otherwise our app updates will not be accepted into the AppStore.

**Comment by kterto** [comment]
Yes, any news on this subject? Since de deadline is approaching, it would be really great to have this fixed in time.

**Comment by thomas-stockx** [comment]
You can just update your pods (or just remove your podfile.lock and run pod install again), afaik this flutter package does not pin to any version of IOSSecuritySuite, so you should automatically get the latest pod (which means 2.x.x).

However, you probably want to avoid this due to https://github.com/jeroentrappers/flutter_jailbreak_detection/issues/61

**Comment by techouse** [comment]
> However, you probably want to avoid this due to https://github.com/jeroentrappers/flutter_jailbreak_detection/issues/61

🤯

Check https://github.com/jeroentrappers/flutter_jailbreak_detection/issues/61#issuecomment-2098606752 for a BSD 2-Clause solution.
